app/views/welcome/_chat.html.erb

```diff
@@ -0,0 +1,19 @@
+<table class= "chat-table">
+  <tr>
+    <th>
+      Chat
+    </th>
+  </tr>
+  <% @chats.each do |chat| %>
+  <tr>
+    <td>
+      <span class="chat-receiver" data-chat-id=<%= chat.id %>>
+        <%= User.find(chat.receiver_id).name %>
```

Comment:
This is not recommended, you are making a query inside a loop. You can try to reach the user from chat (you can use includes in the @chats query or you can create a map [chat.id, receiver_id]